news/2019-10-23-new-featured-artist-false-noise.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "New Featured Artist: False Noise"
+date: 2019-10-22 06:00:00 +0000
+---
+
+We're proud to welcome underground producer **False Noise** as our latest addition to the Featured Artist roster!
+
+![](https://assets.ppy.sh/artists/52/header.jpg)
+
+Rising to prominence in osu! during early 2017 via the [*Skyshards*](https://osu.ppy.sh/beatmapsets/606995) collaborative map by [Shiirn](https://osu.ppy.sh/users/465126) and [Mir](https://osu.ppy.sh/users/8688812), **False Noise** has been something of a cult sensation among the mapping scene for his incredibly unique and compelling musical flavour, fusing elements of drum and bass and stylistic licks from all across electronica, coupled together with a mas
+
+We've managed to snag **11** tracks from his wide catalogue of sound, including *Skyshards*, *Serpentine* and *Kek*, bringing people the **False Noise** they know and love, with a sprinkling of extra tracks as well.
+
+Take a listen for yourself and see what all the hype's about:
+
+### Scarlet Red
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/52/previews/1528.mp3" type="audio/mpeg">
+</audio>
+
+### Shiver
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/52/previews/1530.mp3" type="audio/mpeg">
+</audio>
+
+### Serpentine
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/52/previews/1529.mp3" type="audio/mpeg">
+</audio>
+
+
+### Emerald Green
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/52/previews/1522.mp3" type="audio/mpeg">
+</audio>
+
+
+### Kek
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/52/previews/1527.mp3" type="audio/mpeg">
+</audio>
+
+-------
+
+As always, the above tracks and all the others [**False Noise's** Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/52) are immediately available for modding and mapping, no fuss, and 100% free.
+
+This now leaves us with this week's newest dilemma: the ominous *hint* of our next release to come. Toy's got the reins on this one, so I'll leave it to him:
+
+*You wake up in a dark room. The pungent stench of blight and mildew fills your lungs as you fiddle around for a light switch.*
```

Comment:
if the italics is just for the quote you should use actual quote syntax instead

Reply:
i'm not sure how our news parser handles blockquotes to be honest

Reply:
welp I'm not either so nvm